simulator/tools/command_line_demo.cpp

```diff
@@ -0,0 +1,223 @@
+#include <gazebo/gazebo_config.h>
+#include <gazebo/transport/transport.hh>
+#include <gazebo/msgs/msgs.hh>
+#include <gazebo/gazebo_client.hh>
+
+#include <jointCmd.pb.h>
+
+#include <iostream>
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include "rofi_hal.hpp"
+
+#define CONNECT 1 // Want to connect to gazebo?
```

Comment:
What is the purpose of the define? Should it be a define?

Reply:
It's a leftover from testing without gazebo - deleted.